sample.exs

```diff
@@ -0,0 +1,36 @@
+Mix.install([
+  {:membrane_core, "~> 0.7.0"},
+  {:membrane_ffmpeg_swresample_plugin, "~> 0.7.1"},
+  {:membrane_file_plugin, "~> 0.6.0"},
+  {:membrane_mp3_lame_plugin, "~> 0.8.0"},
+  {:membrane_portaudio_plugin, "~> 0.7.0"}
+])
```

Comment:
let's remove the versions so that the most recent ones were installed

Reply:
> let's remove the versions so that the most recent ones were installed

Done in eee5f58

> and remove the empty test.txt file

Already removed in 2e0a195